engine/class-methods-without-function-keyword.yay

```diff
@@ -0,0 +1,14 @@
+$(macro: recursion) {
+    $(\Pre\Standard\Parser\visibilityModifiers())
+    $(T_STRING as name)
+    (
+    $(optional(\Pre\Standard\Parser\arguments()))
+    )
+    $(optional(chain(token(':'), \Pre\Standard\Parser\type())) as type)
+    $(between(token('{'), layer(), token('}')) as body)
+} >> {
+    $(visibilityModifiers) function $(name) ($$(\Pre\Standard\Expander\arguments($(arguments)))) $$(\Pre\Standard\Expander\type($(type)))
```

Comment:
This may need the `\Pre\Standard\Expander\returnType()` expander that I added at the same time. That'll add the missing `:`, here...